openfisca_france/tests/formulas/irpp.yaml

```diff
@@ -952,3 +952,20 @@
     f4ba: 150000
   output_variables:
     irpp: -48036
+- name: "IRPP - Célibataire ayant des 	Revenu net imposable de 16000 € (teste la décote)"
+  period: 2014
+  absolute_error_margin: 0.9 #TODO : check why the error margin is so large, is there bottom round made by dgfip ?
+  input_variables:
+    salaire_imposable: 16000
+  output_variables:
+    irpp: -183
+    decote: 476
+#TODO : check where the 24.5 difference in IRPP comes from.
```

Comment:
Ok then I'll wait and rebase till parameters haven't been actualized. I'm not sure that I want to investigate on that difference for now since I do not have any real clue. 

If you have a specific task to do or a specific point to check, I'm willing to do it.